### PR TITLE
[CX-592]: feature(submissions/offers/consignments): Add artist filter

### PR DESCRIPTION
**Pull request by amy-mkrtchyan**
Screen Shot 2021-04-02 at 01 26 53
Screen Shot 2021-04-02 at 01 27 02
Screen Shot 2021-04-02 at 01 27 10
Screen Shot 2021-04-02 at 01 27 36
Screen Shot 2021-04-02 at 01 27 48
Screen Shot 2021-04-02 at 01 27 59
Screen Shot 2021-04-02 at 01 28 38
Screen Shot 2021-04-02 at 01 28 47
Screen Shot 2021-04-02 at 01 28 55

- add artist filter
- show selected term in search-box
- make search-box work with state/assigned_to filters

Refs [CX-592]

[CX-592]: https://artsyproduct.atlassian.net/browse/CX-592

### CODE REVIEW

**Comment by bhoggard** [comment]
@amy-mkrtchyan can you add a screenshot?

File: app/views/admin/consignments/index.html.erb [hunk 1]
```diff
@@ -7,10 +7,13 @@
 <div class='container double-padding-top'>
   <div class='row'>
     <div class='row col-md-12'>
-      <%= text_field_tag 'term', term, class: 'form-control', placeholder: 'Search by ID, user name, or partner', id: 'consignments-search-form' %>
+      <%= text_field_tag 'term', display_term, class: 'form-control', placeholder: 'Search by ID, user name, partner or artist', id: 'consignments-search-form' %>
```
Comment (bhoggard) [inline app/views/admin/consignments/index.html.erb:10]: minor: I love Oxford commas, so please add a comma after "partner".

Comment (amy-mkrtchyan) [reply app/views/admin/consignments/index.html.erb]: added :)

**Review by bhoggard — APPROVED** [review]
Great job with code _and_ testing!

**Pull request merged**